.github/workflows/deploy-production.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         uses: descope/descopecli/.github/actions/import@v0.8.2
         with:
           project_id: ${{ vars.PRODUCTION_PROJECT_ID }}
-          management_key: ${{ secrets.MANAGEMENT_KEY }}
+          management_key: ${{ secrets.PROD_MANAGEMENT_KEY || secrets.MANAGEMENT_KEY }}
```

Comment:
Let's rename `PROD_MANAGEMENT_KEY` to `PRODUCTION_MANAGEMENT_KEY` to match the same format as the projectId.

Reply:
Done